kiwi/partitioner/gpt.py

```diff
@@ -116,15 +116,15 @@ def set_flag(self, partition_id: int, flag_name: str) -> None:
 
     def set_uuid(self, partition_id: int, uuid: str) -> None:
```

Comment:
Let's change this function name. That way it's clear what it is.

Reply:
Sorry but this would change the public interface and I'm not very keen on creating a deprecation decorator. Also this name is part of the partitioner interface and exists for msdos, gpt, dasd and has a different meaning according to the different table types anyway. It's a UUID, and what it technically means we can document but I don't think you are able to come up with any name that could not be misinterpreted. So let's stick with the simplest: It's a UUID

Reply:
Okay.